.travis.yml

```diff
@@ -1,16 +1,27 @@
-language: python
+language: php
 
-python:
-    - "2.7"
+php: 5.6
 
-sudo: false
+sudo: true
 
-install: 
-    - "pip install -q -r requirements.txt --use-mirrors"
+cache:
+  directories:
+    - "$HOME/.composer/cache"
+    - _build
 
-script: sphinx-build -nW -b html -d _build/doctrees . _build/html
+before_install:
+  - python --version
+  - sudo pip install --upgrade pip
+
+install:
+  - sudo pip install sphinx==1.1.3
```

Comment:
use ``pip install --user sphinx==1.1.3`` to install packages in the user directory rather than in places requiring root permissions, and you won't need to keep using sudo.